addons/explosives/stringtable.xml

```diff
@@ -508,26 +508,31 @@
         <Key ID="STR_ACE_Explosive_Module_DisplayName">
             <English>Explosive System</English>
             <Polish>System ładunków wybuchowych</Polish>
+            <German>Sprengstoffsystem</German>
         </Key>
         <Key ID="STR_ACE_Explosive_RequireSpecialist_DisplayName">
             <English>Require specialists?</English>
             <Polish>Wymagaj specjalistów?</Polish>
+            <German>Benötigt Experten?</German>
```

Comment:
Regardless of the polish and english translation I definitely prefer somthing like Sprengstoffexperte